composer/workflows/unit_testing_cycle.py

```diff
@@ -30,6 +30,5 @@
         'composer_sample_cycle',
         schedule_interval=datetime.timedelta(days=1),
         default_args=default_dag_args) as dag:
-    start = dummy_operator.DummyOperator(task_id='oops_a_cycle')
-    end = dummy_operator.DummyOperator(task_id='oops_a_cycle')
-    start >> end
+    start = dummy.DummyOperator(task_id='oops_a_cycle')
+    start >> start
```

Comment:
What's going on here?

Reply:
There was an error in the initial code where the error that was being thrown was actually a "duplicate task ID" error, and not a cycle, which is what we wanted to demonstrate. This code actually shows a cycle in the dag, where "start" is dependent on itself to start, so it will never begin